MPM/mpm.php

```diff
@@ -115,7 +117,7 @@ function mpm_webhook()
 				$order->payment_complete();
 			}
 		}
-		elseif ($payment->isOpen() === FALSE)
+		elseif ($payment->isOpen() === FALSE || $onHold)
```

Comment:
Deze `$onHold` check is volgens mij niet nodig. De order moet alleen geupdate worden als de betaling niet meer `OPEN` is. Wanneer de webhook nu wordt aangeroepen wanneer de order `on-hold` is maar de payment nog steeds `OPEN` zal de order status `failed` krijgen, wat niet de bedoeling is.